### ISSUE TITLE
Custom icons for Tabular, Image, Classical-ML, Object Detection, etc

### ISSUE THREAD

**Issue by karkir0003** [DSGT-DLP/Deep-Learning-Playground#776]
**Describe the solution you'd like**
Display custom icons/symbols for tabular, image, classical ml, object detection data source types for a trainspace.

**Additional context**
use your creativity when choosing the icon

**Setup Instructions (what branch to work off of)**
Run the following commands
```
git checkout nextjs
git pull origin nextjs
git checkout -b custom-icon-trainspace-datasource
```
FYI: If you are not able to immediately run `git checkout nextjs`, make sure you commit your changes in the current branch or run `git stash` and then execute the above commands

**Comment by codingwithsurya** [comment]
Royalty Free Marketplace: https://fontawesome.com/icons/file-csv?f=classic&s=solid

**Comment by karkir0003** [comment]
@codingwithsurya status on this?

**Comment by codingwithsurya** [comment]
> @codingwithsurya status on this?

i think this is something you wanted me to do, and then we pivoted to something else. I can def do this, but i think u wanted me to tackle the feedback task first.

**Comment by karkir0003** [comment]
The feedback form revamp? @codingwithsurya

**Comment by codingwithsurya** [comment]
> The feedback form revamp? @codingwithsurya

yea

**Comment by karkir0003** [comment]
you can take on that project if you'd like

**Comment by karkir0003** [comment]
i wrote up design doc, so take a look and execute

**Comment by codingwithsurya** [comment]
> you can take on that project if you'd like

sure. only thing is that this week is pretty busy, i'll only start being active again in DLP in about a week or so.

**Comment by codingwithsurya** [comment]
so not sure about the urgency of it, but if its fine to wait a little bit, i can def do it

**Comment by karkir0003** [comment]
sure. I think feedback form revamp is higher priority

**Comment by codingwithsurya** [comment]
ofc. so i'll start working on feedback form revamp in about a week or so, alright?

**Comment by karkir0003** [comment]
yup. sounds good. feedback form revamp doc linked in this issue: [#906 ](https://github.com/DSGT-DLP/Deep-Learning-Playground/issues/906)

**Comment by karkir0003** [comment]
@codingwithsurya

**Comment by codingwithsurya** [comment]
> yup. sounds good. feedback form revamp doc linked in this issue: [#906 ](https://github.com/DSGT-DLP/Deep-Learning-Playground/issues/906)

got it. thanks.
